### PR TITLE
remove outdated gscholar parser and directly point to gscholar instead

### LINKED ISSUE #6223: Fix the cited_mne.py script to update "Publications by users" page
In our docs, we link to a [Publications by users](http://martinos.org/mne/stable/cited.html) page, which is being created by our [`cited_mne.py`](https://github.com/mne-tools/mne-python/blob/master/doc/sphinxext/cited_mne.py) script, authored by @jasmainak (see https://github.com/mne-tools/mne-python/issues/1935 and https://github.com/mne-tools/mne-python/pull/2954 )

This script has not been run in a long time and according to @larsoner, it's not trivial to get it to work again (python 2 and several dead dependencies).

Some volunteer should overhaul `cited_mne.py` so that we can update our **Publications by users** and show the world how great MNE-Python is.

(the alternative is a stale page in the docs, last updated 2017)

I have not yet found a trivial workflow used by others ... or a package that can easily be tweaked / copied over.

cc: @jona-sassenhagen who tried this before

### PR DESCRIPTION
**Pull request by sappelhoff**
closes #6223

### CODE REVIEW

**Comment by jona-sassenhagen** [comment]
Of course you'll be doing this in the year I'm adding 5 papers to this :|

LGTM otherwise.

File: doc/cited.rst [hunk 1]
```diff
@@ -5,195 +5,8 @@
 Publications by users
 =====================
 
-Papers citing MNE (194) as extracted from Google Scholar (on February 03, 2017).
+Papers citing MNE (482) or MNE-Python (322) as extracted from Google Scholar
+(on April 25, 2019).
```
[the remark this answers is not in the document]
Comment (jona-sassenhagen) [reply doc/cited.rst]: https://scholar.google.com/scholar?oi=bibs&cites=12188330066413208874 :)

[the remark this answers is not in the document]
Comment (sappelhoff) [reply doc/cited.rst]: scroll down to the bottom of the changes :-)

[the remark this answers is not in the document]
Comment (sappelhoff) [reply doc/cited.rst]: it's already there

File: doc/cited.rst [hunk 2]
```diff
@@ -5,195 +5,8 @@
...
-188. `The Temporal Structure of Memory Retrieval1. <http://act-r.psy.cmu.edu/wordpress/wp-content/uploads/2016/11/meg21.pdf>`_. JR Anderson,JP Borst, JM Fincham,A Ghuman… - act-r.psy.cmu.edu.
-189. `Règles de sélection de variables pour accélerer la localisation de sources en MEG et EEG sous contrainte de parcimonie. <http://www.josephsalmon.eu/papers/gretsi2015.pdf>`_. O FERCOQ,A GRAMFORT,J SALMON- josephsalmon.eu.
-190. `Neuroplasticity in Human Alcoholism: Studies of Extended Abstinence with Potential Treatment Implications George Fein1, 2 and Valerie A. Cardenas1  …. <http://www.nbresearch.com/PDF/2014/Neuroplasticity%20in%20Human%20Alcoholism-%20Studies%20of%20Extended%20Abstinence%20with%20Potential%20Treatment%20Implications_Fein%20G,%20Cardenas%20V.pdf>`_. G Fein, AMP Center - nbresearch.com.
+- `MNE <https://scholar.google.de/scholar?cites=12188330066413208874&as_sdt=2005&sciodt=0,5&hl=en>`_
+- `MNE-Python <https://scholar.google.de/scholar?cites=1521584321377182930&as_sdt=2005&sciodt=0,5&hl=en>`_
```
Comment (sappelhoff) [inline doc/cited.rst]: @agramfort @jona-sassenhagen

**Comment by sappelhoff** [comment]
rendered version: https://12654-1301584-gh.circle-artifacts.com/0/dev/cited.html

**Pull request merged**